<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1110</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多页准直器MLC控制系统</string>
  </property>
  <widget class="QWidget" name="HorizontalLayout">
   <layout class="QVBoxLayout" name="verticalLayout_5" stretch="30,1">
    <item>
     <widget class="QWidget" name="mainWindge" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,1">
       <item>
        <widget class="QWidget" name="leftWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,1">
          <item>
           <widget class="QGroupBox" name="fileInfo">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox::title {  
    color: rgb(0, 47, 167); /* 将红色替换为你想要的字体颜色 */  
    font-size: 30px; /* 可选：设置字体大小 */  
    font-weight: bold; /* 可选：设置字体粗细 */  
}</string>
            </property>
            <property name="title">
             <string>文件信息</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="planName">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>计划名称：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="patientName">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>患者姓名：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="recordIndex">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>病历序号：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="treatMode">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>治疗模式：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="machineIndex">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>治 疗 机：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="machineAngle">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>机架角度：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLCAngle">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>准直器角：</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_X1">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>准直器X1:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_X2">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>准直器X2:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_Y1">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>准直器Y1:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_Y2">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>准直器Y2:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QLabel" name="planNameValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="patientNameValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="recordIndexValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="treatModeValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="machineIndexValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="machineAngleValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLCAngleValue">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_X1Value">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_X2Value">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_Y1Value">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="MLC_Y2Value">
                 <property name="autoFillBackground">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="viewList">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox::title {  
    color: rgb(0, 47, 167); /* 将红色替换为你想要的字体颜色 */  
    font-size: 30px; /* 可选：设置字体大小 */  
    font-weight: bold; /* 可选：设置字体粗细 */  
}</string>
            </property>
            <property name="title">
             <string>射野列表</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,10">
             <item>
              <widget class="QWidget" name="widget" native="true"/>
             </item>
             <item>
              <widget class="QListWidget" name="listWidget"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="viewSee">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QGroupBox::title {  
    color: rgb(0, 47, 167); /* 将红色替换为你想要的字体颜色 */  
    font-size: 30px; /* 可选：设置字体大小 */  
    font-weight: bold; /* 可选：设置字体粗细 */  
}</string>
            </property>
            <property name="title">
             <string>射野预览</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="centerWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8" stretch="10,1">
          <item>
           <widget class="QWidget" name="showPicWidget" native="true">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0">
            <item>
             <widget class="QWidget" name="runInfoWidge" native="true">
              <property name="styleSheet">
               <string notr="true">QWidget#runInfoWidge {  
border: 0.5px solid gray; /* 灰色边框，厚度为1像素（注意原注释中的厚度描述与实际代码不符，已更正） */  \nbox-shadow: 0 4px 8px rgba(0, 0, 0, 0.3); /* 添加阴影效果 */  \nbackground-color: rgba(255, 255, 255, 0.2); /* 背景色为带有20%透明度的白色 */
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,4,14">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>√</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>√</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>√</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>√</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>启动设备初始化</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>设备初始化完成</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>启动设备运行</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>设备运行完成</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QLabel" name="picLable">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="picWidget" native="true">
              <property name="font">
               <font>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">border: 0.5px solid gray; /* 灰色边框，厚度为1像素（注意原注释中的厚度描述与实际代码不符，已更正） */  
box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3); /* 添加阴影效果 */  
background-color: rgba(255, 255, 255, 0.2); /* 背景色为带有20%透明度的白色 */</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,10">
          <item>
           <widget class="QWidget" name="topWidge" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="tableWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5,2,2,0,2">
       <item>
        <widget class="QLabel" name="softVersion">
         <property name="text">
          <string>多页准直器控制系统 Standard 2014</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>638</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="userNameLabel">
         <property name="text">
          <string>账号：AleSmart</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="LimitLabel">
         <property name="text">
          <string>权限：Admin</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="runtimeLabel">
         <property name="text">
          <string>运行时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="runtime">
         <property name="text">
          <string>00:00:00</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1110</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="openFile"/>
    <addaction name="openFolder"/>
    <addaction name="separator"/>
    <addaction name="clickExit"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="controllerMenu">
    <property name="title">
     <string>控制</string>
    </property>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="controllerMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="openFolder">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="clickExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
